crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/service/ProsecutionConcludedServiceTest.java

```diff
@@ -80,16 +81,20 @@ void test_whenMultipleProsecutionMessagesAndFlagIsFalseOrTrue_thenProcessingVali
         when(courtDataAPIService.retrieveHearingForCaseConclusion(any())).thenReturn(getWQHearingEntity());
         when(courtDataAPIService.isMaatRecordLocked(any())).thenReturn(false);
         when(offenceHelper.getTrialOffences(any(), anyInt())).thenReturn(List.of(getOffenceSummary("123")));
+        when(caseConclusionDTOBuilder.build(any(), any(), any(), any()))
+            .thenReturn(ConcludedDTO.builder()
+                .prosecutionConcluded(getProsecutionConcluded())
+                .build());
 
         prosecutionConcludedService.execute(getProsecutionConcluded());
 
         //then
         verify(prosecutionConcludedValidator).validateRequestObject(any());
         verify(courtDataAPIService, atLeast(1)).retrieveHearingForCaseConclusion(any());
         verify(courtDataAPIService, atLeast(1)).isMaatRecordLocked(anyInt());
-        verify(prosecutionConcludedImpl, atLeast(1)).execute(any());
+        verify(prosecutionConcludedImpl, atLeast(1)).execute(any(), any());
```

Comment:
Please see previous comment about the use of verify, the majority of these checks can be removed as the tests will fail if the corresponding stub is not invoked.

Reply:
I'm not sure what the original intent of the `verify` statements was here, I agree that the test will fail if the corresponding stub is not invoked (unless somewhere along the line we change away from `strict` mode without realising the consequences). Could the original author here have been actually wanting to ensure that all of these individual services were invoked?

Reply:
I think it's probably just a case of not fully understanding how Mockito works, but let's leave this for now, there are a few other improvements we could do with making so we can tackle this under a separate ticket.